### PR TITLE
`<AnalyticsSummaryCard />` -  add additional testkit methods

### PR DESCRIPTION
**Pull request by adamg-wix**
### 🔦 Summary

Enhance driver + add tests + fix missing propType

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_7071/

**Comment by env-github-token** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_7071

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js [hunk 1]
```diff
@@ -13,5 +57,48 @@ export const analyticsSummaryCardDriverFactory = (base, body) => {
       await base.hover();
       await findByHook(base, dataHooks.analyticsSummaryCardCTA).click();
     },
+    /**
+     * Check if CTA exists
+     * @returns {boolean}
+     */
+    CTAExists: async () => {
```
Comment (mykas) [inline packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:64]: What is CTA?

Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:64]: call to action , main action button of the component.

Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:64]: This is how it was named i was just following convention.

Comment (mykas) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:64]: yeah I see its problematic. Ok, lets move on with this for now

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js [hunk 2]
```diff
@@ -1,10 +1,54 @@
...
+      base.$(`[data-hook="${dataHooks.analyticsSummaryCardTitleTooltip}"]`),
+      body,
+    );
+
+  const getTrendIndicator = () =>
+    trendIndicatorDriverFactory(
+      findByHook(base, dataHooks.analyticsSummaryCardTrend),
+    );
+
+  const getTooltipText = async tooltip => {
```
Comment (mykas) [inline packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:36]: No need for this full blown function. `getTooltipText` already does that for you in wix-ui-core

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js [hunk 3]
```diff
@@ -13,5 +57,48 @@ export const analyticsSummaryCardDriverFactory = (base, body) => {
...
+    /**
+     * returns the title text
+     * @returns {Promise<string>}
+     */
+    title: () => getTitle().getText(),
+    /**
+     * returns the title tooltip text
+     * @returns {Promise<string>}
+     */
+    titleTooltip: async () => getTooltipText(await getTitleTooltip()),
```
Comment (mykas) [inline packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:92]: Lets change this to: `getTitleTooltipText`

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js [hunk 4]
```diff
@@ -13,5 +57,48 @@ export const analyticsSummaryCardDriverFactory = (base, body) => {
...
+    /**
+     * returns the value text
+     * @returns {Promise<string>}
+     */
+    value: () => getValue().getText(),
+    /**
+     * returns the value tooltip text
+     * @returns {Promise<string>}
+     */
+    valueTooltip: async () => getTooltipText(await getValueTooltip()),
```
Comment (mykas) [inline packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:82]: lets change this to `getValueTooltipText`

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js [hunk 5]
```diff
@@ -13,5 +57,48 @@ export const analyticsSummaryCardDriverFactory = (base, body) => {
...
+    /**
+     * Checks whether the Analytics are loading
+     * @returns {Promise<boolean>}
+     */
+    isLoading: () => getLoader().exists(),
+    /**
+     * returns the value text
+     * @returns {Promise<string>}
+     */
+    value: () => getValue().getText(),
```
Comment (mykas) [inline packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:77]: Lets change this to `getValue`

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js [hunk 6]
```diff
@@ -13,5 +57,48 @@ export const analyticsSummaryCardDriverFactory = (base, body) => {
...
+    /**
+     * checks if trend exists
+     * @returns {Promise<boolean>}
+     */
+    trendExists: () => getTrendIndicator().exists(),
+    /**
+     * returns the trend value
+     * @returns {Promise<number>}
+     */
+    trendvalue: () => getTrendIndicator().getTrendValue(),
```
Comment (mykas) [inline packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:102]: Lest change this to `getTrendValue`

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js (hunk 2 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:36]: so it should just return await tooltip.getTooltipText(); ?

Comment (mykas) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js:36]: yep

Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js]: done

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js (hunk 5 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js]: done

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js (hunk 4 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js]: done

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js (hunk 3 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js]: done

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js (hunk 6 above)
Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.js]: done

File: packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.d.ts [hunk 7]
```diff
@@ -1,5 +1,13 @@
 import { BaseUniDriver } from 'wix-ui-test-utils/unidriver';
 
 export interface AnalyticsSummaryCardUniDriver extends BaseUniDriver {
+  CTAExists(): Promise<boolean>;
   clickCTA(): Promise<void>;
+  isLoading(): Promise<boolean>;
+  title(): Promise<string>;
```
Comment (mykas) [inline packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.d.ts:7]: Don't forget to change types too according to driver

Comment (adamg-wix) [reply packages/wix-style-react/src/AnalyticsSummaryCard/AnalyticsSummaryCard.uni.driver.d.ts]: done

**Review by mykas — APPROVED** [review]
Nicely done!

**Pull request merged**